### ISSUE TITLE
'mu project add service migrations' fails

### ISSUE THREAD

**Issue by ajuvercr** [mu-semtech/mu-cli#18]
Running command `mu project add service migrations` results in "Could not fetch image description.  Are you online?".

After looking through mu this is due to [globbing](https://mywiki.wooledge.org/glob) in bash when curling at line 194.
Adding `-g` flag fixes this issue.

Next I was greated with "Could not find image in the repository.", [the search](https://info.mu.semte.ch/microservice-revisions?filter[microservice][:exact:title]=migrations&filter[:exact:version]=latest&page[size]=1) results in an empty data array.
This might be a new problem in the backend.
Possible fix: 
  Change line 177 `serivce_image=$4` to `service_image=semtech/mu-$4-service` 
  Change line 194 to ``service_image_description=`curl -gs "https://info.mu.semte.ch/microservice-revisions?filter[:exact:i
    mage]=$service_image&filter[:exact:version]=$service_tag&page[size]=1"` ``
  
(this probably defeats the purpose though?)

**Comment by erikap** [comment]
@ajuvercr Thanks for the issue report!

The first error "Could not fetch image description. Are you online?" should be fixed in the new v1.0.3 release. It was caused by some local development URLs that slipped through in the released script.

The search currently doesn't support partial/fuzzy search, but only exact matches. Your proposed solution `service_image=semtech/mu-$4-service` will work for your case, but not all service names follow that naming pattern. It's also the intention to support images which are not part of the `semtech` namespace where image naming is out of our control.

If you source the completions in your shell as explained in [the README](https://github.com/mu-semtech/mu-cli#installation) with the latest release, you should have working tab completion now for the `mu project add service` command. That will at least hint that `migrations` is not a valid name, but `mu-migrations-service` should be used instead. It also completes version numbers after the service name btw.

The completions are a bit of an orphaned child at the moment which need to be revisited, but I hope the fixes in v1.0.3 unblock you at least.
